<launch>
   	<node name="viewer1" pkg="lsd_slam_viewer" type="viewer" output="screen">
			<remap from="/lsd_slam/liveframes" to="/lsd1/lsd_slam/liveframes" />
	        <remap from="/lsd_slam/keyframes" to="/lsd1/lsd_slam/keyframes" />
	        <remap from="/lsd_slam/graph" to="/lsd1/lsd_slam/graph" /> 
	</node>	
	
  	<node name="viewer2" pkg="lsd_slam_viewer" type="viewer" output="screen">
			<remap from="/lsd_slam/liveframes" to="/lsd2/lsd_slam/liveframes" />
	        <remap from="/lsd_slam/keyframes" to="/lsd2/lsd_slam/keyframes" />
	        <remap from="/lsd_slam/graph" to="/lsd2/lsd_slam/graph" />
	</node>
		 
</launch>
